cfme/intelligence/reports/schedules.py

```diff
@@ -148,33 +160,20 @@ def __str__(self):
     report_filter = attr.ib()
     active = attr.ib(default=None)
     timer = attr.ib(default=None)
-    from_email = attr.ib(default=None)
-    emails = attr.ib(default=None)
+    email = attr.ib(default=None)
     email_options = attr.ib(default=None)
 
-    @property
-    def exists(self):
-        schedules = self.appliance.db.client["miq_schedules"]
-        return self.appliance.db.client.session\
-            .query(schedules.name)\
-            .filter(schedules.name == self.name)\
-            .count() > 0
-
     def update(self, updates):
         view = navigate_to(self, "Edit")
         changed = view.fill(updates)
         if changed:
             view.save_button.click()
+            view.flash.assert_no_error()
```

Comment:
Why only assert this when the form is saved? I think its good to keep this outside of this `if/else` block so that its always asserted.